src/uproot/source/http.py

```diff
@@ -563,10 +563,24 @@ def __init__(self, file_path, **options):
         self._fallback_options["num_workers"] = self._num_fallback_workers
         self._open()
 
+    def is_lite_notebook(self):
+        try:
+            shell = get_ipython().__class__.__name__
+            # True if running in a jupyter lite notebook
+            # False if running in a jupyter noteboook ('ZMQInteractiveShell') or IPython ('TerminalInteractiveShell')
+            return shell == "Interpreter"
+        except NameError:
+            return False  # Python interpreter
```

Comment:
I wonder whether we should use 
```suggestion
        return sys.platform == 'emscripten'
```

This would not detect IPython (or notebooks), but I think we specifically care about the emscripten part more than IPython?

Reply:
It would have the same result, the function would only return `True` if it is running in a  jupyter lite environment.

Reply:
Thank you for the suggestion, I swapped out the function with only this check as it will be less code.